ddtrace/constants.py

```diff
@@ -48,6 +48,7 @@
 ERROR_TYPE = "error.type"  # a string representing the type of the error
 ERROR_STACK = "error.stack"  # a human readable version of the stack.
 
+COMPONENT = "component"
```

Comment:
is `component` a public-facing concept? Are users expected to interact with it themselves?

Reply:
Just verified and according to our spec it is not supposed to be customer facing, so I just removed this constant declaration and just changed to the tag to be set with the "component" string instead of the constant.